woocommerce/payment-gateway/apple-pay/class-sv-wc-payment-gateway-apple-pay-orders.php

```diff
@@ -49,188 +49,49 @@ class SV_WC_Payment_Gateway_Apple_Pay_Orders {
 	 */
 	public static function create_order( \WC_Cart $cart ) {
 
-		// ensure totals are fully calculated by simulating checkout in WC 3.1 or lower
-		// TODO: remove this when WC 3.2+ can be required {CW 2017-11-17}
-		if ( ! defined( 'WOOCOMMERCE_CHECKOUT' ) && SV_WC_Plugin_Compatibility::is_wc_version_lt( '3.2' ) ) {
-			define( 'WOOCOMMERCE_CHECKOUT', true );
-		}
-
 		$cart->calculate_totals();
 
 		try {
 
 			wc_transaction_query( 'start' );
```

Comment:
just checking - are these `wc_transaction_query()` calls still relevant? I don't see them used in core anymore.

Reply:
I think we can remove the calls to  `wc_transaction_query()` now. They made more sense when we were trying to save every item as it was created, but as you said, they are not used in the core order creation process anymore. I don't see them anywhere else in the latest version of the code either.

Update in f0ae221.